lci-odoo10/odoo/addons/lci_ecole/models/prof.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 lorenzo6ioti
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class Prof(models.Model):
+    _name="ecole.prof"
+    _inherit = 'ecole.prof'
+    salaire=fields.Float(string="Salaire mensuel", compute="_calcSal", store=True)
```

Comment:
When using a float --> check the decimal precision just as Float into other odoo addons
naming
english